src/components/pricing/select-plan.tsx

```diff
@@ -42,7 +42,83 @@ const SelectPlan: React.FunctionComponent = ({children}) => {
                       Discord access
                     </p>
                   </li>
-                  <li className="mt-5 flex items-start lg:col-span-1 lg:mt-0">
+                  <li className="flex items-start lg:col-span-1">
+                    <div className="flex-shrink-0">
+                      <svg
+                        className="h-5 w-5 text-green-400"
+                        xmlns="http://www.w3.org/2000/svg"
+                        viewBox="0 0 20 20"
+                        fill="currentColor"
+                      >
+                        <path
+                          fillRule="evenodd"
+                          d="M10 18a8 8 0 100-16 8 8 0 000 16zm3.707-9.293a1 1 0 00-1.414-1.414L9 10.586 7.707 9.293a1 1 0 00-1.414 1.414l2 2a1 1 0 001.414 0l4-4z"
+                          clipRule="evenodd"
+                        />
+                      </svg>
+                    </div>
+                    <p className="ml-3 text-sm leading-5 text-gray-700 dark:text-gray-100">
+                      Access to all 192 premium courses
```

Comment:
we've got considerably more than this

Reply:
Yeah, we were still crunching the numbers. Does this look right?

Paid courses: 214
Free courses: 44

(I just realized that's not "considerably more")

We also considered using # of lessons instead as it would be BIGGER.

Reply:
I dunno, I don't think it's gonna make a difference wrt to conversions or anything. You could probably just say "over 200" or whatever and it'd be fine instead of hyper-specific which is gonna be wrong quickly unless loaded dynamically at build time.

If "premium" in this instance means "paywalled" 212 sounds about right, not including any courses that are actually playlists.